### PR TITLE
update for JuMP v0.14 and Julia v0.5

### LINKED ISSUE #67: Typo in README
In your README you link to `https://github.com/IainNZ/JuMPeR.jl/blob/master/examples` but this does not exist since your folder is named `example`without `s`.

### LINKED ISSUE #68: Update to JuMP 0.14
Are you planning an update of JuMPeR compatible with JuMP 0.14?

### PR DESCRIPTION
**Pull request by yeesian**
Closes #67 and #68. Summary of Changes:
- [depwarn] `UTF8String` -> `String`
- [depwarn] remove `utf8()`
- [macro] introduction of a new `symvarname` to deal with "variable not defined"
- [macro] added a `JuMP.registercon()` stub for uncset constraints to work with `JuMP.@constraint`

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1,2 @@
-julia 0.4
-JuMP 0.13.2 0.14
+julia 0.5
+JuMP 0.14
```
Comment (IainNZ) [inline REQUIRE]: Can you put a defensive upper bound too?

File: src/operators.jl [hunk 2]
```diff
@@ -222,8 +222,6 @@ const ADP = "an adaptive variable"
 (-)(lhs::UncExpr, rhs::Variable) = UncVarExpr([rhs],[UncExpr(-1)],lhs)
 (*)(lhs::UncExpr, rhs::Variable) = (*)(rhs,lhs)
 (/)(lhs::UncExpr, rhs::Variable) = error("Cannot divide $UNE by a variable")
-# UncExpr--GenericNormExpr
-(/){P,C,V<:Uncertain}(lhs::UncExpr, rhs::GenericNormExpr{P,C,V}) = error("Cannot divide $UNE by $UNM")
```
Comment (IainNZ) [inline src/operators.jl]: ?

File: REQUIRE (hunk 1 above)
Comment (IainNZ) [inline REQUIRE]: Can you put upper bound too?

File: src/operators.jl [hunk 3]
```diff
@@ -269,8 +267,6 @@ const ADP = "an adaptive variable"
 (-)(lhs::UncVarExpr, rhs::Variable) = UncVarExpr(vcat(lhs.vars,rhs),vcat(lhs.coeffs,UncExpr(-1)), lhs.constant)
 (*)(lhs::UncVarExpr, rhs::Variable) = error("Cannot multiply $UVE by a variable")
 (/)(lhs::UncVarExpr, rhs::Variable) = error("Cannot divide $UVE by a variable")
-# UncVarExpr--GenericNormExpr
-(/){P,C,V<:Uncertain}(lhs::UncVarExpr, rhs::GenericNormExpr{P,C,V}) = error("Cannot divide $UVE by $UNM")
```
Comment (IainNZ) [inline src/operators.jl]: ?

**Comment by IainNZ** [comment]
Thanks for doing this!

File: src/operators.jl (hunk 3 above)
Comment (yeesian) [inline src/operators.jl]: dup of https://github.com/yeesian/JuMPeR.jl/blob/39abc7d5ff820c800b1fac62429b4c722b08f074/src/operators.jl#L282

File: src/operators.jl (hunk 2 above)
Comment (yeesian) [inline src/operators.jl]: dup of https://github.com/IainNZ/JuMPeR.jl/blob/abcf0a6efa08ac97311737fe0d43bfd21f44754d/src/operators.jl#L236

**Comment by IainNZ** [comment]
Will tag now

**Comment by IainNZ** [comment]
Actually,  can you tag?

**Pull request merged**